nix/hm-module.nix

```
@@ -0,0 +1,63 @@
+self: {
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (builtins) toString;
+  inherit (lib.types) str int package;
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption;
+  inherit (lib.meta) getExe;
+
+  cfg = config.services.hypridle;
+in {
+  options.services.hypridle = {
+    enable = mkEnableOption "Hypridle, Hyprland's idle daemon";
+
+    package = mkOption {
+      description = "The hypridle package";
+      type = package;
+      default = self.packages.${pkgs.stdenv.system}.hypridle;
```

```suggestion
      default = self.packages.${pkgs.stdenv.hostPlatform.system}.hypridle;
```
Only the latter works when aliases are disabled.